scripts/benchmark/checks/erc20_interact.py

```diff
@@ -71,18 +73,12 @@ def __init__(self, num_blocks: int = 100, num_tx: int = 2) -> None:
         self.addr1 = Web3.toChecksumAddress(FUNDED_ADDRESS)
         self.addr2 = Web3.toChecksumAddress(SECOND_ADDRESS)
 
-    @property
-    @abstractmethod
-    def name(self) -> DefaultStat:
-        raise NotImplementedError(
-            "Must be implemented by subclasses"
-        )
-
-    @abstractmethod
```

Comment:
@williambannas I made `_setup_benchmark` non-abstract because not every child needs that hook.